lib/Dialect/Btor/IR/BtorOps.cpp

```diff
@@ -301,15 +302,15 @@ static ParseResult parseNDStateOpOp(OpAsmParser &parser, OperationState &result)
 
 template <typename Op>
 static LogicalResult verifyArrayOp(Op op) {
```

Comment:
This is a red flag. It would be worth focusing on BitVectors for your cleaned up pr